### PR TITLE
Update GNOME clients

### PR DESCRIPTION
**Pull request by albfan**
- Change bugzilla to gitlab
- Include Polari IRC client

### CODE REVIEW

File: content/components/index.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ This page enlists various components that together make the Telepathy project. F
 ## Clients
 
 * [KDE-Telepathy](http://community.kde.org/KTp) - a collection of components providing IM capabilities to the Plasma desktops
-* [Empathy](http://live.gnome.org/Empathy) - Gnome IM/voice/video client
+* [Empathy](http://live.gnome.org/Empathy) - GNOME IM/voice/video client
```
Comment (Kaffeine) [inline content/components/index.md]: https://wiki.gnome.org/action/show/Apps/Empathy

**Review by Kaffeine — CHANGES_REQUESTED** [review]
Hi Alberto,
thank you for the PR. While you're changing the line with Empathy can you please adjust its URL too?

File: content/resources/debugging.md [hunk 2]
```diff
@@ -5,12 +5,11 @@ categories: ["information"]
 
 ## If you are using Empathy or KDE-Telepathy...
 
-Empathy and KDE-Telepathy both have debug consoles which show you the debug output from components of your choice. Refer to the [Empathy wiki](http://live.gnome.org/Empathy/Debugging) and the [KDE Community wiki](https://community.kde.org/KTp/FAQ#Providing_debug) respectively for more details.
+Empathy and KDE-Telepathy both have debug consoles which show you the debug output from components of your choice. Refer to the [Empathy wiki](/https://wiki.gnome.org/Apps/Empathy/Debugging) and the [KDE Community wiki](https://community.kde.org/KTp/FAQ#Providing_debug) respectively for more details.
 
 If you're not using one of these clients, or you're debugging a crash, or you're just feeling old-school, read on!
 
 ## Connection managers
-
```
Comment (Kaffeine) [inline content/resources/debugging.md]: All other sections have such empty line, so please keep it here too. Other than that — LGTM. :+1:  I'll merge the PR once this latest nitpick fixed.

**Review by Kaffeine — CHANGES_REQUESTED** [review]

File: content/resources/debugging.md (hunk 2 above)
Comment (albfan) [reply content/resources/debugging.md]: Done. I accidentally edit that and reverted incorrectly, :wink:

**Review by Kaffeine — APPROVED** [review]

**Comment by Kaffeine** [comment]
Merged. Thank you!

**Pull request merged**